src/main/kotlin/study/racingcar/generator/RandomNumberGenerator.kt

```diff
@@ -0,0 +1,13 @@
+package study.racingcar.generator
+
+import kotlin.random.Random
+
+class RandomNumberGenerator : Generator {
+    override fun generate(): Int {
+        return Random.nextInt(0, MAXIMUM_NUMBER)
+    }
+
+    companion object {
+        private const val MAXIMUM_NUMBER = 9
```

Comment:
```suggestion
        return Random.nextInt(0..MAXIMUM_NUMBER)
    }

    companion object {
        private const val MAXIMUM_NUMBER = 9
```

<img width="739" alt="image" src="https://user-images.githubusercontent.com/43440195/202446913-30c09714-9a73-4e58-968e-3133f42a24d9.png">
Random.nextInt(from, until)은 위와 같아요. until이기 때문에 끝 구간이 포함되지 않겠네요. 지금과 같은 코드는 양 끝 구간에 대한 정보가 없기 때문에 헷갈리기 쉬운 것 같아요.

`(0..9)`나 `(0 until 10)`처럼 IntRange를 사용한다면 양 구간 포함에 대한 정보를 조금 더 분명하게 나타낼 수 있습니다.

Reply:
(0..9).random()을 활용하였습니다.